UICollectionViewLeftAlignedLayout/UICollectionViewLeftAlignedLayout.m

```diff
@@ -57,7 +57,14 @@ - (NSArray *)layoutAttributesForElementsInRect:(CGRect)rect {
 }
 
 - (UICollectionViewLayoutAttributes *)layoutAttributesForItemAtIndexPath:(NSIndexPath *)indexPath {
-    UICollectionViewLayoutAttributes* currentItemAttributes = [[super layoutAttributesForItemAtIndexPath:indexPath] copy];
+    UICollectionViewLayoutAttributes* currentItemAttributes;
+
+    if (UIDevice.currentDevice.systemVersion.floatValue < 10.0) {
```

Comment:
Would prefer `if (@available(iOS 10, *)) {` be used here.

However, are there any detrimental effects of using `copy` all the time? @petropavel13

Reply:
Hi!
Added available check.
There is no issues detected since app was launched in store.